components/ctas/Cta.vue

```diff
@@ -1,8 +1,11 @@
 <template>
   <component
-    :is="tag"
-    :class="{ button: true, 'button--secondary': secondary }"
-    :to="to"
+    :is="isInternal(to) ? 'nuxt-link' : 'a'"
+    :class="[ 'button', { 'button--secondary': secondary } ]"
+    :href="to"
```

Comment:
I know this is superflual for the `nuxt-link` component but I would prefer no to add it at all. Could you use `isExternal(to) || isMail(to) ? to : null` similar to other `a`-related properties?

Reply:
I tried to do this before but there is a problem when setting `href` to `null` if it's an internal link: it affects the `nuxt-link` component as well, since it's rendered into `a` when it's compiled by Nuxt. So the result is an `a` without `href` at all... and we don't want that.
![Captura de pantalla 2019-10-29 a las 15 07 00](https://user-images.githubusercontent.com/8567677/67774510-cd9c8400-fa5d-11e9-902f-17794e0679fd.png)

Reply:
This is the actual way of how `nuxt-link` is rendered on the DOM now, which I believe it's fine.
![Captura de pantalla 2019-10-29 a las 15 14 15](https://user-images.githubusercontent.com/8567677/67775021-a1353780-fa5e-11e9-8791-a3d3ce7c9926.png)

Reply:
oh! ok. Nevermind.